docs/Homebrew-linuxbrew-core-Maintainer-Guide.md

```diff
@@ -194,21 +194,21 @@ Conflicts:
 
 The `merge-homebrew` command will create a pull-request for you, using `hub`.
 
-It is expected that CI checks on the merge commit of the PR will fail.
-This is due to a bug with Azure Pipelines and its handling of merge
-commits. Master branch builds also fail for the same reason. This is
-OK.
-
-Once the PR is approved by other Homebrew developers, you can finalise
-the merge with:
+Continuous integration checks that the pull request passes
+`brew readall` and `brew style`, which only takes a few minutes.
+Finalise the merge with:
 
 ```bash
-brew pull --clean <PR-NUMBER>
 git push origin master
 ```
 
 The merge is now complete. Don't forget to update your GitHub fork by
-running `git push your-fork master`
+running `git push your-fork master`.
+
+If someone else has pushed to Homebrew/linuxbrew-core before you finish
+the merge, you can update your branch with `git rebase --rebase-merges`,
```

Comment:
I think this paragraph can be moved to before the paragraph above ("The merge is now complete"). If we move this paragraph, we can also slightly improve the wording:

```suggestion
If the above command fails (for example, if another maintainer pushed their changes to Homebrew/linuxbrew-core before you finished the merge),
you can update your branch with `git rebase --rebase-merges`,
```